octoprint_mrbeam/software_update_information.py

```diff
@@ -229,14 +229,15 @@ def switch_software_channel(plugin, channel):
     Returns:
         None
     """
+    _logger.debug("switch_software_channel")
     old_channel = plugin._settings.get(["dev", "software_tier"])
     if channel in software_channels_available(plugin) and channel != old_channel:
         _logger.info("Switching software channel to: {channel}".format(channel=channel))
         plugin._settings.set(["dev", "software_tier"], channel)
         reload_update_info(plugin)
 
 
-def reload_update_info(plugin):
+def reload_update_info(plugin, clicked_by_user=False):
```

Comment:
Not sure why we have `clicked_by_user` and _user_checks_for_updates why not use one of them ?